staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```diff
@@ -38,6 +38,11 @@ import (
 type SecureServingOptions struct {
 	BindAddress net.IP
 	BindPort    int
+	// BindNetwork is the type of network to bind to - defaults to "tcp", accepts "tcp",
+	// "tcp4", and "tcp6".
+	BindNetwork string
+	// Listener is the secure server network listener.
```

Comment:
this still needs docs that either BindAddress+BindPort+BindNetwork are set **or** Listener.

Reply:
OK